test/models/journeyDomain/PreRequisitesDomainSpec.scala

```diff
@@ -160,9 +165,68 @@ class PreRequisitesDomainSpec extends SpecBase with Generators {
           )
         }
 
-      }
+        "when not using the same country of dispatch in Post Transition" - {
+          "goods transport to another country after transit" in {
+            val userAnswers = emptyUserAnswers
+              .setValue(DeclarationTypePage, TIR)
+              .setValue(SameUcrYesNoPage, false)
+              .setValue(SameCountryOfDispatchYesNoPage, false)
+              .setValue(AddCountryOfDestinationPage, OptionalBoolean.yes)
+              .setValue(TransportedToSameCountryYesNoPage, true)
+              .setValue(ItemsDestinationCountryPage, itemsDestinationCountry)
+              .setValue(ContainerIndicatorPage, OptionalBoolean.yes)
+
+            val expectedResult = PreRequisitesDomain(
+              ucr = None,
+              countryOfDispatch = None,
+              countryOfDestination = Some(OptionalBoolean.yes),
+              itemsDestinationCountry = Some(itemsDestinationCountry),
+              containerIndicator = OptionalBoolean.yes
+            )
+
+            val result = PreRequisitesDomain.userAnswersReader(mockPostTransitionPhaseConfig).apply(Nil).run(userAnswers)
+
+            result.value.value mustBe expectedResult
+            result.value.pages mustBe Seq(
+              SameUcrYesNoPage,
+              SameCountryOfDispatchYesNoPage,
+              AddCountryOfDestinationPage,
+              TransportedToSameCountryYesNoPage,
+              ItemsDestinationCountryPage,
+              ContainerIndicatorPage
+            )
+          }
 
-      "when not TIR type in Post transition" in {}
+          "goods NOT transport to another country after transit" in {
+            for (selection <- Seq(OptionalBoolean.no, OptionalBoolean.maybe)) {
```

Comment:
Consider using `forAll(Gen.oneOf(OptionalBoolean.no, OptionalBoolean.maybe))` here so we are informed which input causes a test failure